x-pack/filebeat/module/fortinet/firewall/_meta/fields.yml

```diff
@@ -0,0 +1,2154 @@
+
+    - name: community
+      type: keyword
+      description: >
+        SNMP Community
+
+    - name: configcountry
+      type: keyword
+      description: >
+        Configuration country
+
+    - name: connection_type
+      type: keyword
+      description: >
+        FortiClient Connection Type
+
+    - name: conserve
+      type: keyword
+      description: >
+        Flag for conserve mode
+
+    - name: constraint
+      type: keyword
+      description: >
+        WAF http protocol restrictions
+
+    - name: contentdisarmed
+      type: keyword
+      description: >
+        Email scanned content
+
+    - name: contenttype
+      type: keyword
+      description: >
+        Content Type from HTTP header
+
+    - name: cookies
+      type: keyword
+      description: >
+        VPN Cookie
+
+    - name: count
+      type: integer
+      description: >
+        Counts of action type
+
+    - name: countapp
+      type: integer
+      description: >
+        Number of App Ctrl logs associated with the session
+
+    - name: countav
+      type: integer
+      description: >
+        Number of AV logs associated with the session
+
+    - name: countcifs
+      type: integer
+      description: >
+        Number of CIFS logs associated with the session
+
+    - name: countdlp
+      type: integer
+      description: >
+        Number of DLP logs associated with the session
+
+    - name: countdns
+      type: integer
+      description: >
+        Number of DNS logs associated with the session
+
+    - name: countemail
+      type: integer
+      description: >
+        Number of email logs associated with the session
+
+    - name: countff
+      type: integer
+      description: >
+        Number of ff logs associated with the session
+
+    - name: countips
+      type: integer
+      description: >
+        Number of IPS logs associated with the session
+
+    - name: countssh
+      type: integer
+      description: >
+        Number of SSH logs associated with the session
+
+    - name: countssl
+      type: integer
+      description: >
+        Number of SSL logs associated with the session
+
+    - name: countwaf
+      type: integer
+      description: >
+        Number of WAF logs associated with the session
+
+    - name: countweb
+      type: integer
+      description: >
+        Number of Web filter logs associated with the session
+
+    - name: cpu
+      type: integer
+      description: >
+        CPU Usage
+
+    - name: craction
+      type: integer
+      description: >
+        Client Reputation Action
+
+    - name: criticalcount
+      type: integer
+      description: >
+        Number of critical ratings
+
+    - name: crl
+      type: keyword
+      description: >
+        Client Reputation Level
+
+    - name: crlevel
+      type: keyword
+      description: >
+        Client Reputation Level
+
+    - name: crscore
+      type: integer
+      description: >
+        Some description
+
+    - name: cveid
```

Comment:
I think the only issue I had here was that since there was no dataset to backup that specific field, I was a bit hesitant to add it directly. Some fortinet modules for example would include an array of CVE's, but ECS do not want that field as an array at this point.